curbs/README.md

```diff
@@ -325,6 +372,96 @@ A Curb Space is represented as a JSON object whose fields are as follows:
 
 [Top][toc]
 
+## Curb Object
+
+Defines individual assets located adjacent to a Curb Space or Curb Zone. Important notes about Curb Objects:
+
+  - Curb Objects can be located anywhere: within, beside, or overlapping with Curb Zones, Spaces, or other Curb Objects
+  - Curb Objects must be related to either a Curb Space or Curb Zone
+  - Curb Objects do not have Curb Policies linked directly to them. Associated Curb Policies can be found by looking at the related Curb Zone (either directly or through the Curb Space)
+  - Unlike Zones and similar to Spaces, Objects may be updated as needed, with a new `curb_object_id` being optionally assigned by the city
+
+A Curb Object is represented as a JSON object whose fields are as follows:
+
+| Name   | Type   | Required/Optional   | Description   |
+| ------ | ------ | ------------------- | ------------- |
+| `curb_object_id` | [UUID][uuid] | Required | The ID of the curb object. |
+| `geometry` | [Point][point] | Required |The spatial location of this curb object location. This can represent the approximate center of the object, or the centroid location of the object, depending on its size and shape. |
+| `curb_zone_id` | [UUID][uuid] | Conditionally Required | The ID of the Curb Zone this object is related to. The geometry of the specified Curb Zone does not need to directly relate to the geometry of this object. Either a Zone or Space ID is required for an Object. |
```

Comment:
The curb_zone_id description should reflect the idea that the object's physical location is nearest to this zone. Similarly with curb_space_id below.

Reply:
Added in latest commit

Reply:
It would be great to also be able to relate the object to a policy id for signs.

Reply:
To @felixir's point, INRIX has a specific implementation underway that relates sign assets to policies

Reply:
Ah so adding a `policy_id` to the curb object defintion? I guess that makes some sense, if the object at a curb zone is a sign that has one description on it and that aligns to just one policy. But most curb zone policies are not defined by just one sign? I'm not sure it's needed to do this, since the zone is connected to a policy already, and this would apply just one type of object, `signage`. Could you give us some use cases and reasons why you would need the `policy_id` field in an Object?

Reply:
Yeah, I think we thought that you could relate the object to the policy through the curb zone. Similar to how events can be related to policies through the curb zone.

Reply:
See the new curb_policy_id  field @felixir @Mu-yi-Zhou @alexdemisch @mschwartzie and give feedback on this